docker/app/Dockerfile

```diff
@@ -12,6 +12,15 @@ RUN wget -q -O /tmp/shellcheck.tar.xz "https://storage.googleapis.com/shellcheck
   && rm /tmp/shellcheck.tar.xz \
   && shellcheck --version
 
+RUN wget -q -O get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
+  && chmod 700 get_helm.sh \
+  && ./get_helm.sh \  
+  && helm version
+
+RUN wget -q https://github.com/instrumenta/kubeval/releases/latest/download/kubeval-linux-amd64.tar.gz \
+  && tar xf kubeval-linux-amd64.tar.gz \
```

Comment:
Remove this tar file after the cp to avoid having it included in the Docker layer since the tar file isn't needed at runtime. E.g. [see how we're doing this for shellcheck](https://github.com/ascoderu/opwen-cloudserver/blob/365b91974534f054c2503ca3a5014ae7e521eb4b/docker/app/Dockerfile#L11-L12)).